applications/fishing-map/src/features/workspace/Workspace.tsx

```diff
@@ -117,7 +118,9 @@ function Workspace() {
   const workspace = useSelector(selectWorkspace)
   const workspaceStatus = useSelector(selectWorkspaceStatus)
   const locationCategory = useSelector(selectLocationCategory)
-  const resourceQueries = useSelector(selectDataviewsResourceQueries)
+  const resourceQueries = resolveResourcesFromDatasetConfigs(
```

Comment:
This is going to end up having a new array every time the component is rendered, so why not having a selector which return this function?